scripts/pre-commit-build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+npm run build
+
+(git diff --quiet Adhan.js Adhan.js.map)
+FILE_STATUS=$?
+
+if [[ $FILE_STATUS -ne 0 ]]; then
+    echo "Adhan.js doesn't contain the latest changes. Please run \"npm run build\" and add Adhan.js and Adhan.js.map to your commit."
```

Comment:
if we turn this into a CICD check there's probably some formatting we can use to make this error appear nicely in github